engine/gamesys/src/gamesys/gamesys.cpp

```diff
@@ -243,13 +243,7 @@ namespace dmGameSystem
                 0, CompModelIterProperties,
                 0);
 
-        REGISTER_COMPONENT_TYPE("meshc", 725, mesh_context,
-                CompMeshNewWorld, CompMeshDeleteWorld,
-                CompMeshCreate, CompMeshDestroy, 0, 0, CompMeshAddToUpdate, 0,
-                CompMeshUpdate, 0, CompMeshRender, 0, CompMeshOnMessage, 0,
-                0, CompMeshGetProperty, CompMeshSetProperty,
-                0, CompMeshIterProperties,
-                0);
+        // prio: 725  comp_mesh.cpp
```

Comment:
Remove the mesh context from the function.

Reply:
Ok. 

There also the m_MeshContext property in struct Engine. I searched for references throughout the codebase. It doesn't seem to be used. I removed that too. It compiles and runs ok.

Reply:
yes